gpu/plchain.cu

```diff
@@ -725,15 +769,39 @@ void finish_stream_gpu(const mm_idx_t *mi, const mm_mapopt_t *opt, chain_read_t*
     plchain_debug_analysis(stream_setup.streams[t]);
 #endif // DEBUG_CHECK
     
-    plchain_backtracking(&stream_setup.streams[t].host_mem,
-                         stream_setup.streams[t].reads, misc, km);
+    // TODO: backtrack multiple pending batches
+    // reset values
+    cudaMemsetAsync(stream_setup.streams[t].dev_mem.d_long_seg_count, 0, sizeof(unsigned int),
+                    stream_setup.streams[t].cudastream);
+    cudaMemsetAsync(stream_setup.streams[t].dev_mem.d_mid_seg_count, 0, sizeof(unsigned int),
+                    stream_setup.streams[t].cudastream);
+    cudaMemsetAsync(stream_setup.streams[t].dev_mem.d_total_n_long, 0, sizeof(size_t),
+                    stream_setup.streams[t].cudastream);
+    seg_t* long_segs = stream_setup.streams[t].long_mem.long_segs;
```

Comment:
No need to reset long seg count here: will reset every time before lauching any kernels.